### DISCUSSION TITLE
External scaler and Interceptor terminating

### DISCUSSION THREAD

**Discussion by ryzam** [kedacore/http-add-on#202] [Q&A]
Hi guys, I have a problem to run keda http-add-on. Success follows to install keda core and keda-add-on but unable to create the external scaler and interceptor . It continues reporting pods terminating as in the image below. Any other step I missed?

![image](https://user-images.githubusercontent.com/597759/123660528-550b4d80-d866-11eb-8f21-9af7b4b486df.png)

**Comment by tomkerkhove** [comment]
Did you install it through Helm or seperately?

**Reply by ryzam** [reply]
Installing using HELM and in the logs showed endpoint 'interceptor-admin' not found

**Reply by tomkerkhove** [reply]
Hm odd, can you share the logs please?

**Reply by arschles** [reply]
@ryzam logs from the operator would also be very helpful - can you share those too?

**Comment by ryzam** [comment]
@arschles / @tomkerkhove , FYI 

[keda-add-ons-http-controller-manager.log](https://github.com/kedacore/http-add-on/files/6745271/keda-add-ons-http-controller-manager.log)
[keda-operator-metrics-apiserver.log](https://github.com/kedacore/http-add-on/files/6745273/keda-operator-metrics-apiserver.log)
[keda-operator.log](https://github.com/kedacore/http-add-on/files/6745274/keda-operator.log)

**Reply by ryzam** [reply]
Hi guys, any update on this issue and do you need other logs?

Tq

**Reply by arschles** [reply]
hi @ryzam sorry about that, I'm in the middle of moving and have been busy with that. I'm looking at this now

**Reply by arschles** [reply]
@ryzam I see nothing out of the ordinary in the logs except for this in the metrics server logs:

```console
E0628 13:28:38.894757       1 webhook.go:202] Failed to make webhook authorizer request: Post "https://10.43.0.1:443/apis/authorization.k8s.io/v1/subjectaccessreviews?timeout=10s": context canceled 
```

It looks like maybe the server can't talk to the auth endpoints. Could RBAC be preventing this?

**Reply by ryzam** [reply]
Ok tq, I will check RBAC settings

**Reply by arschles** [reply]
Thanks @ryzam !

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
